Tasks/AzureRmWebAppDeployment/operations/PublishProfileUtility.ts

```diff
@@ -0,0 +1,53 @@
+import tl = require('vsts-task-lib/task');
+import { TaskParameters } from './TaskParameters';
+import fs = require('fs');
+import * as Constant from './Constants';
+
+var packageUtility = require('webdeployment-common/packageUtility.js');
+var parseString = require('xml2js').parseString;
+
+export class PublishProfileUtility {
+
+    private static async GetPublishProfileJsonFromFile(publishProfilePath: string): Promise<any> {
+        return new Promise((response, reject) => {
+            var pubxmlFile = packageUtility.PackageUtility.getPackagePath(publishProfilePath);
+            var publishProfileXML = fs.readFileSync(pubxmlFile);
+            parseString(publishProfileXML, (error, result) => {
+                if(!!error) {
+                    reject(error);
```

Comment:
Can we append some user-friendly message here?

Reply:
done